src/components/SVGIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import IconUSBC from '../../assets/USBC.svg'
+import IconEdit from '../../assets/Edit.svg'
+import IconWifi from '../../assets/Wifi.svg'
+import IconIot from '../../assets/IoT.svg'
+import IconLaptop from '../../assets/Laptop.svg'
+
+interface SVGIconProps {
+    type: string;
+}
+
+const iconTypes = {
+  iconUSBC: IconUSBC,
+  iconEdit: IconEdit,
+  iconWifi: IconWifi,
+  iconIot: IconIot,
+  iconLaptop: IconLaptop,
+};
+
+const SVGIcon = ({ type, ...props }) => {
```

Comment:
Please use interfaces. I suggest creating an interface that extends whatever interface the props of SVGIcon uses. Also, ideally we'd use a different name for the SVGIcon variable and component.

Reply:
Got it! What is a good name instead of SVGIcon?